src/runtime/markdown-parser/frontmatter.ts

```diff
@@ -1,6 +1,8 @@
-import matter from 'gray-matter'
```

Comment:
Regardless of playground discussion, I believe parser is normally used in server and can be moved to `runtime/server/markdown-parser` for code clarity.

Reply:
Actually I think we should have the ability to make it run on client-side as well (ideally through a web worker)

Reply:
What would be the use-case and benefits?